### PR TITLE
GWY-3927: Custom Events Rules API specs

### PR DESCRIPTION
**Pull request by vova07**
Here is the right diff: https://github.com/Rebilly/RebillyAPI/compare/rules...GWY-3927

### CODE REVIEW

**Comment by vova07** [comment]
Please review!

File: spec/definitions/CustomRules/CustomRuleSet.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    description: Custom event description
+    readOnly: true
+  version:
+    type: integer
+    readOnly: true
+  rules:
+    type: array
+    items:
+      $ref: "#/definitions/Rule"
+  updatedTime:
```
Comment (XaosSintez) [inline spec/definitions/CustomRules/CustomRuleSet.yaml]: no `createdTime`, right?

File: spec/definitions/Rules/Actions/pick-gateway-account.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+description: ""
```
Comment (XaosSintez) [inline spec/definitions/Rules/Actions/pick-gateway-account.yaml]: Some descriptions would be nice, i think

File: spec/definitions/Schedule/ScheduleInstruction.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+type: object
+description: "The calculation instruction of scheduled time"
+discriminator: method
+properties:
+  method:
+    type: string
```
Comment (XaosSintez) [inline spec/definitions/Schedule/ScheduleInstruction.yaml]: may be set default to `auto`?

File: spec/definitions/CustomRules/CustomRuleSet.yaml (hunk 1 above)
Comment (vova07) [inline spec/definitions/CustomRules/CustomRuleSet.yaml]: I think it should be.
I'll add it. 
Thanks!

**Comment by vova07** [comment]
I think I need to create a new PR without `slava`'s commits.

**Comment by vova07** [comment]
The new PR: https://github.com/Rebilly/RebillyAPI/pull/43

File: spec/definitions/Schedule/ScheduleInstruction.yaml (hunk 3 above)
Comment (adamaltman) [inline spec/definitions/Schedule/ScheduleInstruction.yaml]: I don't think so -- I believe `auto` is something special.

File: spec/paths/custom-rules.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+get:
+  tags:
+    - Custom Rules
+  summary: Retrieve a list of rules sets
```
Comment (adamaltman) [inline spec/paths/custom-rules.yaml]: doesn't this retrieve a list of only the custom rules sets?

Comment (adamaltman) [inline spec/paths/custom-rules.yaml]: or rather custom event rules sets?

**Comment by adamaltman** [comment]
It is missing the tracking for custom events too?

File: spec/paths/custom-rules.yaml (hunk 4 above)
Comment (vova07) [inline spec/paths/custom-rules.yaml]: Yes, you are right, but I thought if we are in `Custom Rules` section, it's a clear thing that we talk only about their rule sets.

**Comment by vova07** [comment]
Yes, `tracking` is miss, because after discussion with Slava, we decide that this is part of another PR.
And we must decide how we'll go with `tracking`. I mean will be it a common `API URL` or we'll go with 2 separate `endpoints`.

**Comment by vova07** [comment]
If is it possible, please comment and review the new PR: https://github.com/Rebilly/RebillyAPI/pull/43
It contains only the file I added.
Thanks!

**Pull request closed without merging**